packages/cactus-plugin-ledger-connector-corda/src/main-server/kotlin/gen/kotlin-spring/src/main/kotlin/org/hyperledger/cactus/plugin/ledger/connector/corda/server/impl/ApiPluginLedgerConnectorCordaServiceImpl.kt

```diff
@@ -346,4 +352,108 @@ class ApiPluginLedgerConnectorCordaServiceImpl(
         logger.info("Returning {} NodeInfo elements in response.", nodeInfoList.size)
         return nodeInfoList
     }
+
+    /**
+     * Start monitoring state changes for stateClass specified in request body.
+     */
+    override fun startMonitorV1(req: StartMonitorV1Request?): StartMonitorV1Response {
```

Comment:
@outSH Recommendation: I'd try to make it work with JVM SocketIO instead of HTTP polling for these reasons:
1. Polling uses more CPU/bandwidth
2. Keep it consistent with the other plugin implementations that we have

Reply:
Yeah, that was my initial plan too (mainly because of #2), but the `socket.io-server-java` setup looked overly complicated for this scenario and time frame, especially for someone like me who did not work at all with spring boot before. It doesn't seem very popular either, I would be very cautious before basing entire monitoring on that. `netty-socketio` was easier to setup and more popular, but unfortunately it doesn't support socketio v4. 

Side note - as described in previous review comment, current polling application is less likely to skip/loose events than socketio implementation. Similar reliability features could be implemented on socketio as well of course, but I'm afraid the implementation would be much more complex than polling one. Final implementation to be decided soon, but I'd rather not use socket.io here for reasons mentioned above.

Reply:
@outSH Cool. Would you mind including this information in the commit message as well? If we try to retrace our steps later on it will be a very useful reminder to people like me who are having trouble remembering simple things like what they had for breakfast. (joking, but not joking)
And I know it's on GH as well, but if we ever migrate, then we'll most likely only be able to take the git commit history not whatever is stored in GH's private database as comments.

Reply:
@petermetz Sure, agree, I've updated the commit message to clarify the monitoring design.

Reply:
@outSH Cheers, resolving this.